sandbox/descriptions.html.example

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Not a comment for this PR, but can we just change all the sandbox files to drop the `.example` from them so they just work out of the box without requiring people to do anything? @heff